.github/test_dependencies.yaml

```diff
@@ -13,7 +13,8 @@ e2e:
     # renovate: datasource=docker depName=kindest/node versioning=docker
     - 'v1.23.17'
   gke:
-    - 'v1.26.5'
+    # renovate: datasource=custom.gke-rapid depName=gke versioning=semver
```

Comment:
What's the `depName` used for? Is it just for naming?

Reply:
Yeah, in the case of custom datasources it's the string that will be used in PRs titles, in the dashboard, etc.